src/drivers/sf1xx/sf1xx.cpp

```diff
@@ -594,24 +593,17 @@ SF10A::cycle()
 		return;
 	}
 
-
-	/* Trigger measurement */
-	if (OK != measure()) {
-		DEVICE_DEBUG("measure error lidar");
-	}
-
```

Comment:
According to datasheet there the only case of stopping measurement update if someone write other that '\0' to device. In case of device reset (due to power failure or firmware bug) we can write '\0' in case of measurement timeout.